tests/executest.py

```diff
@@ -40,7 +40,11 @@ def _json_object_hook(d):
     for job in joblist:
         if job.jobid != args.jobid: continue
         if args.description: print("{descr}".format(**job._asdict())) 
-        if args.command:     print("root4star -b -q -l \'bfc.C({nevents}, \"{chopts}\", \"{inppath}/{inpfile}\")\'".format(**job._asdict()))
+        if args.command:
+            if job.inpfile=="":
+                print("root4star -b -q -l \'bfc.C({nevents}, \"{chopts}\")\'".format(**job._asdict()))
+            else:
+                print("root4star -b -q -l \'bfc.C({nevents}, \"{chopts}\", \"{inppath}/{inpfile}\")\'".format(**job._asdict()))                
```

Comment:
```suggestion
        if args.command:
            if job.inpfile:
                print("root4star -b -q -l \'bfc.C({nevents}, \"{chopts}\", \"{inppath}/{inpfile}\")\'".format(**job._asdict()))
            else:
                print("root4star -b -q -l \'bfc.C({nevents}, \"{chopts}\")\'".format(**job._asdict()))
```

Reply:
Do these minor suggestion look good? If so, let's accept and merge the PR

Reply:
> Do these minor suggestion look good? If so, let's accept and merge the PR

Yes, they look good to me.